unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -27,6 +30,8 @@ def search_content_chunks(
         search_string: str,
         search_type: ContentSearchType,
         limit: int,
+        reranker: Optional[RerankerConfig] = None,
+        fts_search_language: str = DEFAULT_FTS_SEARCH_LANGUAGE,
```

Comment:
I would like either to remove the prefix fts_ or renaming the parameter to full_text_search_language so it is easy to understand for others. In my opinion search_language would be sufficient.